docs/index.md

```diff
@@ -10,11 +10,24 @@ description: |-
 
 The provider can be used to interact with [Juju][0] - a model-driven Operator Lifecycle Manager (OLM) by Canonical.
 
-Initially this provider allows you to:
+The provider only interacts with a single controller.
 
-* Manage models,
-* Manage applications and deploy charms from CharmHub,
-* Manage integrations.
+Today this provider allows you to manage the following via resources:
+
+* Applications and deploy charms from CharmHub
+* Credentials for existing clouds.
```

Comment:
I'd drop the period as we don't use it anywhere else in the list.

Reply:
done